tests/gnrc_dhcpv6_client/Makefile

```diff
@@ -39,10 +39,13 @@ TEST_ON_CI_BLACKLIST += all
 
 include $(RIOTBASE)/Makefile.include
 
-.PHONY: dhcpv6_server
+.PHONY: dhcpv6_server sudo
 
-dhcpv6_server:
-	$(CURDIR)/dhcpv6_server.sh $(DHCPV6_SERVER_PORT) $(CURDIR)/kea-dhcp6.conf
+dhcpv6_server: sudo
```

Comment:
The `sudo` dependency is needed as the script goes directly to background, obscuring the password prompt. I am aware that this won't work on systems with sudo rules, where any sudo call requires a password input, but I think it is better than just letting the script in background fail due to missing sudo permissions.